mslib/utils/mssautoplot.py

```diff
@@ -109,7 +112,7 @@ def load_from_operation(op_name, msc_url, msc_auth_password, username, password)
     session.auth = msc_auth
     session.headers.update({'x-test': 'true'})
     # ToDp fix config_loader it gets a list of two times the entry
-    response = session.get(urljoin(msc_url, 'status'), timeout=tuple(config_loader(dataset="MSCOLAB_timeout")[0]))
+    response = session.get(urljoin(msc_url, 'status'), timeout=MSCOLAB_TIMEOUT[0])
```

Comment:
Why does it only take the first element here, unlike in the other places?

Reply:
I have no idea, I had that once changed to the common, but decided to keep the value until that question is looked up.

Reply:
This is a bug https://github.com/Open-MSS/MSS/issues/2730 the ToDo was a hint. 

I try to write a test for that first.